bfvmm/src/vmcs/test/test.h

```diff
@@ -39,6 +39,10 @@ class vmcs_ut : public unittest
 
 private:
 
+    ///////////////////////////////////////////////////////////////////////////
```

Comment:
get rid of the comments. group all of the tests by file. See here for example:
https://github.com/Bareflank/hypervisor/blob/master/bfvmm/src/memory_manager/test/test.h